src/main/webapp/app/controllers/submission/submissionListController.js

```diff
@@ -216,7 +214,7 @@ vireo.controller("SubmissionListController", function (NgTableParams, $controlle
 
         var resetBatchUpdateStatus = function () {
             $scope.advancedfeaturesBox.processing = false;
-            $scope.advancedfeaturesBox.assignee = findFirstAssignable();
```

Comment:
Originally the 0th user might not be "assignable", has that changed?

Reply:
this has changed. all the users in the list are now only assignable users.